### DISCUSSION TITLE
readme 中的交流渠道需要更新

### DISCUSSION THREAD

**Discussion by wineee** [linuxdeepin/developer-center#3377] [General | 常规讨论]
1. 根据 #2842，可以加入 matrix 群链接
2. 由于众所周知的一些事，Freenode 早已不是 irc 的首选，希望转到  Libera.Chat

**Comment by BLumia** [comment]
README 会统一改一次，新的（最小基准）模板是 https://github.com/linuxdeepin/developer-center/wiki/README-Template 这个，里面只导向官网，论坛和 developer-center 了。
